src/main/views/user-profiles/create-user-form.html

```diff
@@ -0,0 +1,136 @@
+                email: true
+            },
+            jurisdictionDropdown: {
+                required: true
+            },
+            caseTypeDropdown: {
+                required: true
+            },
+            stateDropdown: {
+                required: true
+            }
+        },
+        messages :{
+            "idamsId" : {
+                required : 'Enter Idams Id',
+                email: 'Email address invalid'
+            },
+            "jurisdictionDropdown": {
+                required : 'Choose a jurisdiction'
+            },
+            "caseTypeDropdown": {
+                required : 'Choose a case type'
+            },
+            "stateDropdown": {
+                required : 'Choose a state'
+            }
+        }
+    });
+
+
+    var listItems= "<option value='' selected>--Select a value--</option>";
+    for (var i = 0; i < jurisdictionLookups.length; i++){
+        listItems+= "<option value='" + jurisdictionLookups[i].id + "'>" + jurisdictionLookups[i].name + "</option>";
+    }
+    $("select#jurisdictionDropdown").html(listItems);
+    $("select#jurisdictionDropdown").on('change',function(){
+        var str = "";
+        $( "select#jurisdictionDropdown option:selected" ).each(function() {
+            updateCaseTypeBasedOn($("select#jurisdictionDropdown")[0].selectedIndex -1);
+      
+        });
+     });
+      $("select#caseTypeDropdown").on('change',function(){
+       var jurisdictionIndex = $("select#jurisdictionDropdown")[0].selectedIndex -1;
+       var caseTypeIndex = $("select#caseTypeDropdown")[0].selectedIndex -1;
+        var listItems= "<option value='' selected>--Select a value--</option>";
+        if(caseTypeIndex > -1){
+            var stateTypes = jurisdictionLookups[jurisdictionIndex].case_types[caseTypeIndex].states;
+ 
+            for (var i = 0; i < stateTypes.length; i++){
+            listItems+= "<option value='" + stateTypes[i].id + "'>" + stateTypes[i].name + "</option>";
+            }
+        }
+         $("select#stateDropdown").html(listItems);
+      });
+  });
+
+</script>
+{%endblock%}
+ {% block content %}
+ {% include "menu.html" %}
+<h2 class="heading-large padding">Create User profile</h2>
+
+<div class="padding">
+    {% if (error)%}
+        <div class="error-summary govuk-error-message">
+            {{error.text}}
+        </div>
+    {% endif %}
+
+    <form id="createuserform" name="createuserform" method="POST">
+    
+      {{ csrfProtection(csrfToken) }}
+        <div class="form-group">
+        
+            <label class="form-label" for="first-name-2">User Idams ID</label>
+            <input class="form-control" id="idamsId" name="idamsId" type="text"><br><br>  
+        <label class="form-label">Live from</label>
+            {{ dateMacro() }}
+            <label class="form-label">Live to</label>
+            {{ dateMacro() }}
+            <label class="form-label" >Work basket default jurisdiction</label>
```

Comment:
The linting might pick up the trailing space before the first `>`, so might be best to remove it.